README.md

```diff
@@ -9,23 +9,159 @@ An engineer working for a [CSE](./CSE.md) project...
 
 This is the playbook. You are invited to contribute via standard git pull requests to GitHub.
 
-## The Basics
-
-* [Source Control](./Engineering/SourceControl.md)
-* [Unit Testing](./Engineering/UnitTesting.md)
-* [Continuous Integration / Continuous Delivery](./Engineering/CICD.md)
-* [Code Reviews](./Engineering/CodeReviews.md)
-* [Retrospectives](./Engineering/Retrospectives.md)
-* [Engineering Practices And Principles](./Engineering/PracticiesAndPrinciples.md)
-
-## Seeking Contributions for
-
-- Component Tests
-- DevOps
-    - [Component Versioning](./Engineering/ComponentVersioning.md)
-    - [Logging and Monitoring](./Engineering/DevOpsLogging.md)
-    - Performance
-    - Telemetry
-- Security
-    - Threat Model
-- Regulatory Compliance
+## Outline
+
+The following outlines each section and what content would be included within each section.
+
+### Goals
+
+The proposed outline is intended to accomplish the following goals
+
+1. Author as idea-to-delivery value stream narrative from an engineering perspective.
+2. Organize for quick reference and discoverability.
+3. Extensible heirarchy to allow teams to share deep subject matter expertise.
+
+### Working Heirarchy
+
+The working heirarchy is rooted in CSE's one week sprint cycle. Each section within then allows for deep dive information, recipes, and use cases.
+
+- [Project Start](0000-pre-sprint)
+  - [Team Agreements](0000-pre-sprint/team-agreements)
+    - [Working Agreements](0000-pre-sprint/team-agreements/working-agreements)
+    - [Definition of Ready](0000-pre-sprint/team-agreements/definition-of-ready)
+    - [Definition of Done](0000-pre-sprint/team-agreements/definition-of-done)
+    - [Estimation ](0000-pre-sprint/team-agreements/estimation)
+  - [Repository organization strategies (How many repositories? How should I decide?)](0000-pre-sprint/source-control-repositories)
+    - Setting up a new repository (readme, license, ignores, etc)
+      - Recipe for ADO
```

Comment:
Somewhat of a fuzzy line, IMO. We include usage-specific guidance for these platforms, but not for Git itself (or in theory, other VC systems)? Perhaps it's better to assume that the reader will know how to use whichever VC platform they intend to use, just as they'll need to know Git (or whatever VC system they use).

We should, however, provide guidance on the semantics of what should be done in terms of merging, branching, etc. from a high-level, but we should assume the reader can learn the "how" of that guidance through officially maintained documentation (i.e. git manpages).

Reply:
Removed these. I believe I added these originally just to show where the recipe's would live. We can figure out what recipes are needed once we finish out the conceptual content.